src/containers/AdminDashboard/AdminDashboard.js

```diff
@@ -339,7 +342,19 @@ const AdminDashboard = (props) => {
     };
 
     const responsesMarkup = useMemo(() => {
-        return questionnaireResponses.map((response, index) => {
+        let filterQuestionnaireResponses;
+        if (searchTerm) {
+            const fuse = new Fuse(questionnaireResponses, {
+                keys: [`questionnaireResponse.${filterBy}`],
+                minMatchCharLength: 3,
+            });
+            filterQuestionnaireResponses = fuse
```

Comment:
what is the fuse for? searching through? we should be able to do that with raw js and not need to add on another library but I might be missing something.

Reply:
if you need help gong through the logic to this feel free to ping me.

Reply:
if we are sure we only want a simple case, exact substring(possibly case insensitive?) matching then the matching would be easy to with js string methods. Fuse would it make it easy to do fuzzy matching (things close to the word). Let me know what you think happy to change it.